.circleci/requirements.txt

```diff
@@ -2,7 +2,7 @@ Cython==0.29.5
 PyCifRW==4.4
 coverage==4.5.4
 ddt==1.2.0
-gsd==1.7.0
-gtar==1.0.1
-numpy==1.16.1
-rowan==1.2.0
```

Comment:
I found it pretty annoying that unit tests would start failing randomly when the requirements are not fixed and easier to manually increase the explicitly tested versions in a specific branch. Removing the versions has the advantage that you will always test the latest version by default, but it also means that some PRs will randomly fail, completely unrelated to the PR. You can go either way, but just wanted to mention that.

Reply:
I recommend reverting this - keeping specific versions is my preference, as well.

Reply:
Fixed. Thanks for the feedback